app/models/comment.rb

```diff
@@ -0,0 +1,16 @@
+class Comment < ApplicationRecord
+  belongs_to :post, counter_cache: true
+  belongs_to :author, class_name: 'User'
+  belongs_to :user
+
+  def update_comments_counter
+    post.comments_counter += 1
+    post.save
+  end
```

Comment:
- [ ] Just like you did in the `Post` model, please use `after_save :update_comments_counter`